packages/calcite-components/src/components/functional/InternalLabel.tsx

```diff
@@ -0,0 +1,61 @@
+import { TemplateResult } from "lit-html";
+import { h } from "@arcgis/lumina";
+
+interface InternalLabelProps {
+  bottomSpacingDisabled?: boolean;
+  centerAlign?: boolean;
+  labelText?: string;
+  onClick?: () => void;
+  required?: boolean;
+  spacingInlineEnd?: boolean;
+  spacingInlineStart?: boolean;
+  tooltipText?: string;
+}
+
+export const CSS = {
+  alignmentEnd: "internal-label-alignment--end",
```

Comment:
Could this naming align more with `centerAlign` - either centerAlign + endAlign, or alignmentEnd and alignmentCenter ?